src/layouts/header/index.tsx

```diff
@@ -20,10 +21,19 @@ type Props = {
 };
 
 const Header = ({ className }: Props) => {
+  const [isSignatured, setIsSignatured] = useState(false);
   const sticky = useSticky();
   const { offcanvas, offcanvasHandler } = useOffcanvas();
   const { search, searchHandler } = useFlyoutSearch();
-  const { connect, connected } = useMetamask();
+  const { connect, connected, sign } = useMetamask();
+
+  useEffect(() => {
+    if (typeof window !== undefined) {
+      const signature = localStorage.getItem('authToken');
+      if (signature) setIsSignatured(true);
```

Comment:
I didn't realize. You can use this like 
`
setIsSignature(!!signature)
`
instead 
`
 if (signature) setIsSignatured(true)....
`